extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js

```diff
@@ -8,14 +8,18 @@ import { api } from 'dicomweb-client';
  * by manually implementing a query option.
  */
 export default class StaticWadoClient extends api.DICOMwebClient {
-  static filterKeys = {
-    "StudyInstanceUID": "0020000D",
-    "PatientName": "00100010",
-    "00100020": "mrn",
-    "StudyDescription": "00081030",
-    "StudyDate": "00080020",
-    "ModalitiesInStudy": "00080061",
-    AccessionNumber: "00080050",
+  static studyFilterKeys = {
+    StudyInstanceUID: '0020000D',
+    PatientName: '00100010',
+    '00100020': 'mrn',
+    StudyDescription: '00081030',
+    StudyDate: '00080020',
+    ModalitiesInStudy: '00080061',
+    AccessionNumber: '00080050',
+  };
+
+  static seriesFilterKeys = {
+    SeriesInstanceUID: '0020000E',
```

Comment:
Can you add SeriesNumber to this list just to be able to test with an extra parameter for the filters.